### PR TITLE
Completed object manipulation exercise - Viviane V.

### PR DESCRIPTION
**Pull request by vivianevieira**
Screen Shot 2020-10-18 at 7 28 39 PM
Screen Shot 2020-10-18 at 7 28 46 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Viviane! Please turn this in again once you have completed all the functions in the challenge :grin:

File: object-manipulation-1/src/get-value.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+
+function getValue(object, key) {
+  var value;
+  for (var property in object) {
+    value = object[key];
+  }
```
Comment (thebearingedge) [inline object-manipulation-1/src/get-value.js:7]: You can do this without a loop. Just access the property `key` on the `object` and return it.

File: object-manipulation-1/src/set-value.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+
+function setValue(object, key, value) {
+  for (var property in object) {
+    object[key] = value;
+  }
```
Comment (thebearingedge) [inline object-manipulation-1/src/set-value.js:6]: You can do this without a loop. Just assign the `value` to the `object` at the property `key`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: object-manipulation-1/src/set-value.js (hunk 2 above)
Comment (vivianevieira) [reply object-manipulation-1/src/set-value.js]: OK! I just revised both files and pushed them.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**